frontend/src/pages/videos/VideoCard.tsx

```diff
@@ -18,12 +21,52 @@ const useStyles = makeStyles(() => ({
 function VideoCardPage() {
   const classes = useStyles();
   const { video_id } = useParams<{ video_id: string }>();
+  console.log(video_id);
+  const video = useVideoMetadata(video_id);
+  console.log(video);
+  const captions: any = video.criteria_scores
+    ? video.criteria_scores.length > 0
+      ? Object.fromEntries(
+          video.criteria_scores.map((criteria) => [
+            criteria.criteria,
+            criteria.criteria,
+          ])
+        )
+      : {
+          d: 'd',
+        }
+    : null;
```

Comment:
This code works, but is a bit hard to read.

Let's make the `VideoCardPage` component easier to maintain by moving the `captions` calculation in a function outside of its body. There are several advantages:
- a human will read and understand what the component is doing faster
- the newly created function could be re-used by many other components

Here is a suggestion you can add in a single click : )

```suggestion
  const captions: any =
    video.criteria_scores?.length > 0 ? getCriteria(video.criteria_scores) : {};
```